### DISCUSSION TITLE
Quickly flip the color of a group of notes

### DISCUSSION THREAD

**Discussion by mcendu** [ppy/osu#29977] [Ideas]
It is potentially useful in the osu!taiko editor to quickly flip the color of a group of notes with a single shortcut, e.g. when copy-pasting long (>2 notes) patterns and seeking variety. I suggest binding the action to <kbd>Ctrl</kbd>+<kbd>J</kbd> ("flip vertically") by default.

**Comment by bdach** [comment]
> I suggest binding the action to Ctrl+J ("flip vertically") by default.

I would not. That'd just be conflating unrelated concepts.
